httpapi/router.go

```diff
@@ -33,12 +29,9 @@ import (
 // @host localhost:8082
 // @schemes http
 func Router(
+	ctx map[string]interface{},
```

Comment:
I don't like the use of `ctx map[string]interface{}` outside of bootstrappers, lets just use the `context.Context` passed to `apiServer.Start` method when referring to the context after server process has started, since its not thread safe afterwards. I know its not being written to but this increases the possibility.

Reply:
We are already using it in api package. It is called synchronously. ATM, I would like to keep it that way

Reply:
I'll change it, seems like we follow this in API already